nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -198,13 +202,22 @@ contract PolicyManager is Upgradeable {
     /**
     * @notice Withdraw reward by node
     **/
-    function withdraw() public {
+    function withdraw() public returns (uint256) {
+        return withdraw(msg.sender);
+    }
+
+    /**
+    * @notice Withdraw reward by node
+    * @param _recipient Recipient of the reward
+    **/
+    function withdraw(address _recipient) public returns (uint256) {
```

Comment:
Interesting - does this mean that `PolicyAuthor`s can withdraw tokens to any `_recipient` address?

Reply:
Yes, @szotov and I talked about this. IIRC, it was easier for him and it made sense. Have a staking address and a receiving address.

Reply:
:+1: